test/config-luatest/sources_test.lua

```diff
@@ -1,8 +1,11 @@
 local t = require('luatest')
+local fun = require('fun')
 local json = require('json')
+local yaml = require('yaml')
 local treegen = require('test.treegen')
 local justrun = require('test.justrun')
 local source_file = require('internal.config.source.file').new()
+local server = require('test.luatest_helpers.server')
```

Comment:
I would prefer to move the tests that require Tarantool to run into test files that already have such tests, or create a new test file. This file was created for the purpose of testing "source" modules only.

Reply:
My idea was to group per topic: source, applier, cli options and so on -- not instruments we're using to implement the test.

I'm OK to move the test case into another file, but how to name it? `sources_execute_test.lua`? Don't know.

The existing test cases in this file also run tarantool processes, but using another helper: `justrun`, not `server`. I can rewrite the new test case in the same way. I guess it is simple enough to be implemented in either way.

Personally I just use what looks simpler in particular case. In one of the next PRs I even wrap server/justrun into a set of more high-level helpers and it feels natural to use them in one test file.